### PR TITLE
Make PG public and add FETP dahsboards to SB

### PR DESCRIPTION
**Pull request by nick-beyondessential**
### Changes:

- add PG to public access country list
- copy FETP dashboards to SB 
- remove STRIVE mapOverlays from explore

### CODE REVIEW

File: packages/database/src/migrations/20210301123426-AddFetpDashboardsToSolomonIs-modifies-data.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    await db.runSql(`
+    select * from "dashboardGroup" where 'fetp' = ANY("projectCodes") and "organisationUnitCode" IN ('PG', 'SB');
+  `)
+  ).rows;
+
+  const sbDashboardGroups = dashboardGroups
+    .filter(r => r.organisationUnitCode === 'SB')
+    .map(dg => dg.code);
+
+  const pgDashboardGroups = dashboardGroups.filter(r => r.organisationUnitCode === 'PG');
```
Comment (edmofro) [inline packages/database/src/migrations/20210301123426-AddFetpDashboardsToSolomonIs-modifies-data.js:32]: Confusing that `pgDashboardGroups` contains the record, whereas `sbDashboardGroups` contains the codes. Would suggest aligning both to have the same data structure, or rename one

File: packages/database/src/migrations/20210301123426-AddFetpDashboardsToSolomonIs-modifies-data.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+          'dashboardGroup',
+          { dashboardReports: dg.dashboardReports },
+          { code: sbCodeToCheck },
+        );
+
+      return insertObject(db, 'dashboardGroup', {
+        organisationLevel: dg.organisationLevel,
+        userGroup: dg.userGroup,
+        organisationUnitCode: 'SB',
+        dashboardReports: `{${dg.dashboardReports.join(',')}}`,
```
Comment (edmofro) [inline packages/database/src/migrations/20210301123426-AddFetpDashboardsToSolomonIs-modifies-data.js:49]: Why is the `.join` required here but not in the `updateValues`?

File: packages/database/src/migrations/20210301123426-AddFetpDashboardsToSolomonIs-modifies-data.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        code: sbCodeToCheck,
+        projectCodes: `{${dg.projectCodes.join(',')}}`,
+      });
+    }),
+  );
+};
+
+exports.down = function (db) {
+  return db.runSql(`
+    update "dashboardGroup" set "dashboardReports" = '{}' where "code" in (${arrayToDbString(
```
Comment (edmofro) [inline packages/database/src/migrations/20210301123426-AddFetpDashboardsToSolomonIs-modifies-data.js:60]: It confuses me that we'd leave dashboard groups with no reports at all in them, but given this is the `down` that won't be run, I'm not too worried

**Review by edmofro — APPROVED** [review]
Looks like it does what we want - just a couple of comments for clarity but nothing holding back approval :-)

File: packages/database/src/migrations/20210301123426-AddFetpDashboardsToSolomonIs-modifies-data.js (hunk 2 above)
Comment (nick-beyondessential) [reply packages/database/src/migrations/20210301123426-AddFetpDashboardsToSolomonIs-modifies-data.js:49]: I don't know why but `updateValues` seems to handle the js array conversion but `insertObject` was throughing a psql malformed array error.

File: packages/database/src/migrations/20210301123426-AddFetpDashboardsToSolomonIs-modifies-data.js (hunk 3 above)
Comment (nick-beyondessential) [reply packages/database/src/migrations/20210301123426-AddFetpDashboardsToSolomonIs-modifies-data.js:60]: Yes just reverting it to the pre-migration state, used mostly while I dev and test and a mild sense of completion.

File: packages/database/src/migrations/20210301123426-AddFetpDashboardsToSolomonIs-modifies-data.js (hunk 1 above)
Comment (nick-beyondessential) [reply packages/database/src/migrations/20210301123426-AddFetpDashboardsToSolomonIs-modifies-data.js:32]: Yep fair call, renamed to sbDashboardGroupCodes

File: packages/database/src/migrations/20210301123426-AddFetpDashboardsToSolomonIs-modifies-data.js (hunk 3 above)
Comment (edmofro) [reply packages/database/src/migrations/20210301123426-AddFetpDashboardsToSolomonIs-modifies-data.js:60]: And the current state is that the groups exist with no reports inside? Weird!

**Review by edmofro — APPROVED** [review]

File: packages/database/src/migrations/20210301123426-AddFetpDashboardsToSolomonIs-modifies-data.js (hunk 3 above)
Comment (nick-beyondessential) [reply packages/database/src/migrations/20210301123426-AddFetpDashboardsToSolomonIs-modifies-data.js:60]: Added when the FETP project was created before any reports created.

**Pull request merged**